doc/en/user/source/services/wfs/webadmin.rst

```diff
@@ -66,6 +66,17 @@ A list of values to be declared in all feature types can be provided in the WFS
 
 The list will be used only for the capabilities document generation. It does not limit the actual target SRS usage in GetFeature requests.
 
+Configuration of Output Format types allowed in GetFeature
+----------------------------------------------------------
+
+Checking the ``Enable Output Format type checking`` checkbox will turn on the enforcement of restrictions on the availability of Output types for GetFeature requests.  It will also limit what Output Format types will be displayed as available in GetCapabilities responses. GetFeature requests with Output types not included in the ``Allowed Output types`` panel will result in ``ForbiddenFormat ServiceException`` response. 
```

Comment:
Less words, same meaning (I hope).

```suggestion
Checking the ``Enable Output Format type checking`` checkbox will enable restrictions on the available output formats for GetFeature requests.  It will also limit what Output Format types will be displayed as available in GetCapabilities responses. GetFeature requests with Output types not included in the ``Allowed Output types`` panel will result in ``ForbiddenFormat ServiceException`` response. 
```